src/WarcraftLegacies.Source/Quests/Dalaran/QuestTheramore.cs

```diff
@@ -28,23 +30,17 @@ public sealed class QuestTheramore : QuestData
       "The distant lands of Kalimdor remain untouched by human civilization. If the Third War proceeds poorly, it may become necessary to establish a forward base there.",
       @"ReplaceableTextures\CommandButtons\BTNHumanArcaneTower.blp")
     {
-      AddObjective(new ObjectiveCapitalDead(dalaran));
+      AddObjective(new ObjectiveResearch(Constants.UPGRADE_R0A7_ESCAPE_TO_THERAMORE_DALARAN, Constants.UNIT_H002_THE_VIOLET_CITADEL_DALARAN_OTHER));
       AddObjective(new ObjectiveSelfExists());
       _rescueUnits = theramoreRect.PrepareUnitsForRescue(RescuePreparationMode.HideNonStructures);
     }
 
     /// <inheritdoc />
     protected override string RewardFlavour =>
-      "With the Violet Citadel destroyed, Jaina leads her people East";
+      "With the once mighty city of Dalaran at the brink of destruction, Jaina leads her people East";
```

Comment:
The research can be done without Dalaran being in any danger so not sure this is really descriptive of the flavour. Also... It's not east, right? Maybe:
```suggestion
      "Jaina Proudmoore abandons the once mighty city of Dalaran and leads her people across the sea, arriving in the untamed lands of Kalimdor.";
```